_posts/2016-09-06-api-hour-22.md

```diff
@@ -50,3 +50,6 @@ inscrire sur la page Eventbrite de cette session: [http://clermontech-apihour-22
 Introduction à RethinkDB, un SGBD NoSQL qui permet de facilement se sharder et se répliquer pour un excellent niveau de service. RethinkDB propose un langage fonctionnel très pratique et propose même de faire des jointures.
 
 ### Claude Dioudonnat • Redis
+
+Avez-vous une table "PeleMele" avec les colonnes "Nom" et "Valeur". Redis a une solution pour faire disparaître cette table honteuse.
```

Comment:
* Wrong punctuation at end of first sentence (question).
* Is `PeleMele` the table name (then it's okay) or the concept (then I'd prefer _pêle-mêle_)?
* What about styling `Nom` and `Valeur` (and `PeleMele` if applicable) rather than using quotes?